### DISCUSSION TITLE
RSS feed

### DISCUSSION THREAD

**Discussion by 4e576rt8uh9ij9okp** [tanujnotes/Olauncher#479] [Ideas]
Hi there, I used the Lunar Launcher before and it has a RSS feed.

I know Olauncher is a "simple af" launcher but being up to date with the interests you have is a good thing I guess.
It's nothing distracting, more like reading news paper.

**Comment by arthaqs** [comment]
There is an RSS opensource app called Feeder. It's very simple, minimalistic AF :) go for it!
